tests/test_codelist.py

```diff
@@ -213,12 +213,24 @@ def test_to_csv():
 
 
 def test_stray_tag_fails():
-    """Check that typos in a tag raises expected error"""
+    """Check that stray brackets from, e.g. typos in a tag, raises expected error"""

```

Comment:
This could be nicely implemented as a pytest-mark to reduce repetition, see https://github.com/IAMconsortium/nomenclature/blob/f5289357e4d58503d3deac9137408940bd9c3e10/tests/test_core.py#L179